generator/AWSPSGeneratorLib/HelpMaterials/Examples/LM/Get-LMAlias.xml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>Get-LMAlias -FunctionName "MylambdaFunction123" -Name "newlabel1" -Select RoutingConfig</code>
```

Comment:
Could you also include the output of the cmdlet?

Reply:
Included the output.